scripts/globals/spells/trust/yoran-oran_uc.lua

```diff
@@ -16,6 +17,31 @@ end
 spellObject.onMobSpawn = function(mob)
     xi.trust.message(mob, xi.trust.messageOffset.SPAWN)
 
+    -- TODO: Nott weaponskill needs implemented and logic added here for Yoran-Oran to use at 50% MP at level 50.
+    -- TODO: UC trusts are supposed to get bonuses depending on unity ranking. Needs research.
+    -- TODO: Custom spawn messages if Unity ranking is higher.
+    -- TODO: Figure out what level of Fast Cast he has.
+
+    mob:addMod(xi.mod.CURE_POTENCY, 50)
+    mob:addMod(xi.mod.REGAIN, 50)
+    mob:addMod(xi.mod.MPP, 15) -- This is supposed to increase with Unity rank, but I don't believe that's implemented so it is set to the minimum.
```

Comment:
This comment looks like a TODO, and should be marked as such

Reply:
Done.